src/utils/common.js

```diff
@@ -0,0 +1,28 @@
+/*  
+    classNames example usage:
+    <div className={classNames(['base-class', {'second' : someFlag, 'third' : true}])}></div>
+*/
+const classNames = (names) => {
```

Comment:
I think it might be convenient to do this:

```
const classNames = (baseClass, ...names) => {
```

then you'd be able to use it as 

```
<div className={classNames('base-class', 'another-class',{'second' : someFlag, 'third' : true})}></div>
```

(without extra array wrapper)